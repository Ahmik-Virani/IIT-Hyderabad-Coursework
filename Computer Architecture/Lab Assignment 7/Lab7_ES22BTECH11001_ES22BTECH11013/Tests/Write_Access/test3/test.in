cache_sim enable config.txt
load input.s
run
cache_sim dump cache.dump
mem 0x10008 1
cache_sim invalidate
mem 0x10008 1
exit