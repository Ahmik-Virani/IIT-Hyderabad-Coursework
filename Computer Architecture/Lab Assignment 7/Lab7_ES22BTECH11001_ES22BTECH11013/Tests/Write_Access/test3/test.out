D-cache statistics: Accesses=8, Hit=3, Miss=5, Hit Rate=0.375
memory[0x10008] = 0x0
memory[0x10008] = 0x10